ddtrace/contrib/pymysql/patch.py

```diff
@@ -9,14 +9,14 @@
 
 from ...ext import db
 from ...ext import net
+from ...internal.schema import schematize_service_name
 from ...internal.utils.formats import asbool
 
 
 config._add(
     "pymysql",
     dict(
-        # TODO[v1.0] this should be "mysql"
```

Comment:
Ha!

I dropped this line because:
1. `v1` actually changes this to 'mysql' (via the schematize_service_name call)
2. Service names will be managed via the schema from now on